torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -906,6 +922,187 @@ def codegen_define(self, kernel: CppTemplateKernel) -> str:
         return result
 
 
+@register_micro_gemm(
+    *generate_gemm_config(
+        VecAVX512,
+        [
+            (4, 32, 64),
+            (8, 32, 64),
+        ],
+        input_dtype=torch.bfloat16,
+        input2_dtype=torch.int8,
+        output_dtype=torch.float,
+        compute_dtype=torch.float,
+        extra_check=check_int8_woq_small_m_dim,
+    ),
+)
+class CppMicroGemmWoQSmallMDim(CppMicroGemm):
```

Comment:
Can we leverage the existing AVX512 micro gemm to add prefetch to support your cases?

Reply:
I added a TODO for it